packages/site/CHANGELOG.md

```diff
@@ -5,12 +5,32 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [0.2.0]
+### Uncategorized
```

Comment:
You need to run `auto-changelog` with the `--prettier` flag, and put the changes into categories